docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:9.6
+    ports:
+      - 5432:5432
+    volumes:
+      - dbdata:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=timed
+      - POSTGRES_PASSWORD=timed
+
+  frontend:
+    build:
+      context: .
+    ports:
+      - 4000:80
```

Comment:
Why 4000? Also, you need to document this

Reply:
Because if we use an override file, ports get merged and there would be a conflict having 4200 assigned twice

Reply:
Removed override, the port is 4200 now